common/app/controllers/EmailSignupController.scala

```diff
@@ -90,16 +90,36 @@ class EmailSignupController(
       Cached(60)(RevalidatableResult.Ok(views.html.emailLanding(emailLandingPage)))
     }
 
+  def renderFooterForm(listName: String): Action[AnyContent] =
+    csrfAddToken {
```

Comment:
Checking, this will have the same behaviour as the existing one, essentially right? Just want to check it won't bust through the cache on every request.

Reply:
Hey Gareth, this is related to the meeting happening on Thursday.
It can be completely ignored for now, happy to chat about it :)